includes/wikia/helpers/FileNamespaceSanitizeHelper.php

```diff
@@ -88,12 +88,8 @@ private function convertToPlainText( $filename ) {
 	private function extractFilename( $potentialFilename, $filePrefixRegex ) {
 		$trimmedFilename = trim( $potentialFilename, "[]" );
 		$unprefixedFilename = mb_ereg_replace( $filePrefixRegex, "", $trimmedFilename );
-		$filenameParts = explode( '|', $unprefixedFilename );
-		if ( !empty( $filenameParts[ 0 ] ) ) {
-			return urldecode( $filenameParts[0] );
-		}
 
-		return null;
+		return self::removeImageParams( $unprefixedFilename );
```

Comment:
`getCleanFileMarkersFromWikitext` expects images in wikitext in form `[[File:xyz.jpg | bla]]` whereas here they may come in the form `File:xyz.jpg` as well as `xyz.jpg` and regex used in getCleanFileMarkersFromWikitext will not work